### DISCUSSION TITLE
How to make PR and merge of changed utilsversion*.*?

### DISCUSSION THREAD

**Discussion by octaeder** [texstudio-org/texstudio#2415] [Q&A] [answered]
utilsversion.cpp, utilsversion.h, and utilsversion_t.h depend on each other. If I make a PR with all of them then the test may fail, since old test utilsversion_t.h runs. If I make a first PR with only utilsversion_t.h than a following PR with the other files should run ok. But any other PR before my second will fail the (now new) test, because not all new files are present.

Could you please clarify if I'm right and how I should proceed. There is no need to rush, because there are some things to consider (create more test data, update some methods with checks of parameters).

Thx

**Comment by sunderme** [comment, marked as answer]
I fail to understand why changing all of said files at once does not work.
txs is build from a complete set of sources and then the test is run. There is no mix of old and new ...

**Reply by octaeder** [reply]
thx
